app/src/main/java/com/amaze/filemanager/ui/activities/MainActivity.java

```diff
@@ -1961,9 +1962,10 @@ public void showSftpDialog(String name, String path, boolean edit) {
       if (i != -1) name = dataUtils.getServers().get(i)[0];
     }
     SftpConnectDialog sftpConnectDialog = new SftpConnectDialog();
-    SshConnectionPool.ConnectionInfo connInfo = new SshConnectionPool.ConnectionInfo(path);
+    FtpConnectionPool.ConnectionInfo connInfo = new FtpConnectionPool.ConnectionInfo(path);
```

Comment:
I'm just wondering why this isn't executed from another thread than the UI thread? This will create some ANR/lag.